profiles/pci/graphic_drivers/profiles.toml

```diff
@@ -22,7 +22,7 @@ class_ids = "0300 0302 0380"
 vendor_ids = "10de"
 device_ids = "*"
 priority = 10
-packages = 'nvidia-utils egl-wayland nvidia-settings opencl-nvidia lib32-opencl-nvidia lib32-nvidia-utils libva-nvidia-driver vulkan-icd-loader lib32-vulkan-icd-loader'
+packages = 'nvidia-utils egl-wayland nvidia-settings opencl-nvidia lib32-opencl-nvidia lib32-nvidia-utils libva-nvidia-driver vulkan-icd-loader lib32-vulkan-icd-loader linux-firmware-nvidia'
```

Comment:
NVIDIA drivers don't need linux-firmware package. They ship GSP firmware themselves with nvidia-utils package. linux-firmware-nvidia package needed only for nouveau driver.